code/about.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Penguins</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <link rel='icon' href='/favicon.ico' type='image/x-icon'>
+  </head>
+  <body>
+
+  <header>
+    <h1>PENGUINS</h1>
+  </header>
+
+  <div class="main">
+    <h2 class="font-header">Impressum</h2>
+    <p>Text Sources: <br>
+      <a href="https://www.penguins-world.com" target="_blank">Penguins-World</a><br>
```

Comment:
I think it is nice with the links and that you use `target="_blank"` to open them in a new window! 👍 

I would recommend to also include `rel="noopener noreferrer"` for each of the links, in this case:
`<a href="https://www.penguins-world.com" target="_blank" rel="noopener noreferrer">`